workflows/reference_files/2.4/reference_files_header.smk

```diff
@@ -75,12 +75,6 @@ for build_name, build_info in config["genome_builds"].items():
     assert "provider" in build_info and genome_provider in possible_providers, (
         f"`provider` not set for `{build_name}` or `provider` not among {possible_providers}."
     )
-    if "genome_fasta_url" in build_info:
```

Comment:
Could you delete the similar lines below starting from `if "capture_bed_url" in build_info:` please?

Reply:
Done 👍